docs/readmes/lte/deploy_install_debian.md

```diff
@@ -1,44 +1,45 @@
 ---
-id: deploy_install_ubuntu
-title: Install AGW
+id: deploy_install_debian
+title: Install AGW on Debian (Stretch)
 hide_title: true
 ---
 
-# Install Access Gateway on Ubuntu (Bare Metal)
```

Comment:
Should we just say that debian install are not supported starting >=1.6 and link to debian 1.5.2 install instead of always copying it the 1.5.2 doc?

Reply:
Agreed. Add this as a note in the ubuntu install and just link -- no need to keep this doc around if we already don't support debian in v1.6